core/crypto.mk

```diff
@@ -47,7 +46,6 @@ $(warning Warning: Enabling CFG_CRYPTO_SHA256 [required by CFG_WITH_PAGER])
 CFG_CRYPTO_SHA256:=y
 endif
 endif
-
```

Comment:
Unrelated change

Reply:
Yes, sorry about that. I will fix it too.